python/ray/_private/worker.py

```diff
@@ -1756,7 +1756,7 @@ def custom_excepthook(type, value, tb):
 
 
 def print_to_stdstream(data):
-    should_dedup = data.get("pid") not in ["autoscaler", "raylet"]
+    should_dedup = data.get("pid") not in ["autoscaler"]
```

Comment:
this means error messages from raylet might be duplicated (e.g., segfault).

Reply:
Yup this seems ok for now. The original message will be in the raylet log files.